todolist/src/components/TodoItem.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
-import { useDispatch } from 'react-redux';
-import { toggleTodo } from '../actions/todoActions';
+//import { useDispatch } from 'react-redux';
+//import { toggleTodo } from '../actions/todoActions';
 import '../css/TodoItem.css';
 
-const TodoItem = ({ todo }) => {
-  const dispatch = useDispatch();
+const TodoItem = ({ todo,onDelete }) => {
```

Comment:
fix code format and configure vs code to format on save

Reply:
Done.